Sloth.Core/Extensions/StringExtensions.cs

```diff
@@ -59,8 +59,15 @@ public static string SafeRegexRemove(this string value, string regEx = @"[^0-9a-
 
         public static string JsonPrettify(this string json)
         {
-            using var jDoc = JsonDocument.Parse(json);
-            return JsonSerializer.Serialize(jDoc, new JsonSerializerOptions { WriteIndented = true });
+            try
+            {
+                using var jDoc = JsonDocument.Parse(json);
+                return JsonSerializer.Serialize(jDoc, new JsonSerializerOptions { WriteIndented = true });
+            }
+            catch (Exception)
```

Comment:
just do `try{ .. } catch { }` if you aren't going to use the exception.  You don't need the param arg.